conftest.py

```diff
@@ -48,6 +48,11 @@ def pytest_cmdline_main(config):
                   .format(opt))
 
 
+# TODO this has to turn into something more sensible
+def pytest_sessionstart(session):
+    store.current_appliance.check_no_conflicting_providers()
```

Comment:
As is, this causes an `INTERNALERROR` which I don't like at all - I'd prefer to just halt collection and print a regular error message but not sure how that's done yet. Thoughts? Ideas?